Dropped support for Django versions prior to 2.2

smithdc1 opened the pull request:
In line with Django's supported versions policy, only Django 2.2 and later is supported. Can therefore remove testing against Django 2.0.

https://www.djangoproject.com/download/

marianoeramirez commented:
Sounds good, but we should update the documentation as well, Can you include the modification of the documentation on this PR?

smithdc1 commented:
@marianoeramirez -- I updated the version on the read me and added a note ahead of the next release. 

Other than these changes I couldn't see much documentation for versions. Am I missing something?

marianoeramirez commented:
Yes, it`s exactly that. Thanks!!
